src/dataDisplay/personPanelSummary.jsx

```diff
@@ -0,0 +1,707 @@
+                contactMethodText = 'Prefers Text';
+
+                if (phone) {
+                    preferredContactInfoText = phone;
+                }
+
+                break;
+            default:
+                if (renderPhone({ phone })) {
+                    preferredContactInfoText = renderPhone({ phone });
+                } else if (renderEmail({ email })) {
+                    preferredContactInfoText = renderEmail({ email });
+                }
+        }
+
+        if (isDoNotEmail || isDoNotMail || isDoNotPhone || isDoNotText) {
+            if (contactMethodText) {
+                contactMethodText += ', DNC via';
+            } else {
+                contactMethodText = 'DNC via ';
+            }
+
+            if (isDoNotEmail) {
+                contactMethodText += 'Email, ';
+            }
+
+            if (isDoNotMail) {
+                contactMethodText += 'Mail, ';
+            }
+
+            if (isDoNotPhone) {
+                contactMethodText += 'Phone, ';
+            }
+
+            if (isDoNotText) {
+                contactMethodText += 'Text';
+            }
+
+            contactMethodText = trimEnd(contactMethodText, ', ');
+        }
+    }
+
+    if (!isChild && (contactMethodText || preferredContactMethod !== 'none')) {
+        return (
+            <React.Fragment>
+                {contactMethodText && (
+                    <Typography
+                        // eslint-disable-next-line react/prop-types
+                        className={classes.preferredContactMethod}
+                        component="h5"
+                        variant="h5"
+                    >
+                        {`(${contactMethodText})`}
+                    </Typography>
+                )}
+
+                {preferredContactMethod !== 'none' && preferredContactInfoText && (
+                    <Typography
+                        // eslint-disable-next-line react/prop-types
+                        className={classes.preferredContactInfo}
+                        variant="body2"
+                    >
+                        {preferredContactInfoText}
+                    </Typography>
+                )}
+            </React.Fragment>
+        );
+    }
+
+    return null;
+}
+
+const useStyles = makeStyles((theme) => {
+    const {
+        palette,
+        shape: {
+            borderRadius,
+        },
+    } = theme;
+    const transitionDuration = '200ms';
+    const colorTransition = `color ${transitionDuration} ease-out`;
+
+    return {
+        root: {
+            [`&.${bemClass}`]: {
+                backgroundColor: palette.background.light,
+                borderRadius,
+                color: palette.text.primary,
+                cursor: 'pointer',
+                outline: 'none',
+                padding: '13px 11px',
+                position: 'relative',
+                transition: colorTransition,
+                userSelect: 'none',
+                '&::before, &::after': {
```

Comment:
Do we have ability to change personal panel summary gradient color via props like same as `infoBar` component ??

Reply:
The color will change via the record type (e.g. `isAdult`) and their gender.

Reply:
```javascript
<PersonPanelSummary
    data={{
        gender: 'm',
        recordType: 'audlt',
    }}
/>
```

Reply:
Is there anything that needs to be done here @fantaJinMode?

Reply:
Seems good, thank you.